requirements.txt

```diff
@@ -19,4 +19,5 @@ prompt-toolkit==2.0.10
 axe_selenium_python==2.1.6
 pytest-snapshot==0.9.0
 beautifulsoup4>=4.12.3
-openai==1.12.0
\ No newline at end of file
+openai==1.12.0
+pytest-asyncio
```

Comment:
Can you please set "pytest-asyncio" version to maintain consistency with other package declaration?

Reply:
I have pinned the `pytest-asyncio` package to `0.23.7` version